packages/babel-plugin-check-es2015-constants/README.md

```diff
@@ -2,10 +2,17 @@
 
 Validate ES2015 constants
 
+<blockquote class="babel-callout babel-callout-info">
```

Comment:
Small note, this block should remain on the website per: https://github.com/babel/babel/pull/4903#discussion_r89925251